genienlp/tasks/generic_task.py

```diff
@@ -378,3 +378,18 @@ def _make_example(self, parts, dir_name=None, **kwargs):
 
     def get_splits(self, root, **kwargs):
         return CrossNERDataset.return_splits(name=self.name, path=root, make_example=self._make_example, **kwargs)
+
+
+@register_task('ood_task')
+class OODTask(BaseTask):
+    def __init__(self, name, args):
+        self.id2label = ['0', '1']
+        self.num_labels = 2
+        super().__init__(name, args)
+
+    @property
+    def metrics(self):
+        return ['recall', 'precision']
```

Comment:
The first metric is the one we use to choose the best model during training, and should probably be F1